job_post_scripts/linkedin_jobpost.py

```
@@ -19,6 +20,85 @@
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+        return response.json()
+    except requests.exceptions.RequestException as e:
+        print(f"Error getting user info: {e}")
+        return None
+
+
+    
+
+def post_job_on_linkedin(text):
+    user = session.get("user")
```

Ruff (0.11.9)

Undefined name `session`

(F821)